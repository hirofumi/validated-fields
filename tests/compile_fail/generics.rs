use validated_fields::ValidatedFields;

#[derive(Debug, PartialEq, ValidatedFields)]
struct Foo<X> {
    x: X,
}
