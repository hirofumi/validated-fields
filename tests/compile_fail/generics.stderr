error: ValidatedFields is not supported for structs with generic parameters
 --> tests/compile_fail/generics.rs:4:1
  |
4 | struct Foo<X> {
  | ^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compile_fail/generics.rs:6:2
  |
6 | }
  |  ^ consider adding a `main` function to `$DIR/tests/compile_fail/generics.rs`
